x-pack/solutions/security/plugins/security_solution/public/common/components/header_section/index.tsx

```diff
@@ -152,7 +155,9 @@ const HeaderSectionComponent: React.FC<HeaderSectionProps> = ({
                       <EuiFlexItem grow={false}>
                         <EuiButtonIcon
                           data-test-subj="query-toggle-header"
-                          aria-label={i18n.QUERY_BUTTON_TITLE(toggleStatus)}
+                          aria-label={`${toggleAriaLabel || ''} ${i18n.QUERY_BUTTON_TITLE(
```

Comment:
when `toggleAriaLabel` is falsy, you will end up with a leading whitespace here. I would do it with `['a', 'b'].join(' ')` instead of inline template string